Bluepill-cli/BPInstanceTests/BPTestHelper.m

```diff
@@ -66,15 +65,21 @@ + (NSString *)samplePhotoPath {
 }
 
 + (NSString *)bpExecutablePath {
-    return [[[[NSBundle bundleForClass:[self class]] bundlePath] stringByDeletingLastPathComponent] stringByAppendingPathComponent:@"bp"];
+    return [[self debugMacOSPath] stringByAppendingPathComponent:@"bp"];
```

Comment:
Don't understand why you need to change this?

Reply:
This is because BluepillLib in no longer is the default `/Debug/Build` directory but inside a `/UninstallXX/Intermediate.buid/` directory